app/src/main/java/com/conupods/OutdoorMaps/View/MapsActivity.java

```diff
@@ -91,19 +88,20 @@ private void initializeMap() {
     public void onMapReady(GoogleMap googleMap) {
         Log.d(TAG, "Map is ready");
 
-        mMap = googleMap;
-        mBuildingOverlays = new BuildingOverlays(mMap, getString(R.string.geojson_url));
-        fusedLocationProvider = LocationServices.getFusedLocationProviderClient(this);
-        mCameraController = new CameraController(mMap, mPermissionsGranted, fusedLocationProvider);
-        mBuildingInfoWindow = new BuildingInfoWindow(getLayoutInflater());
+        BuildingOverlays mBuildingOverlays = new BuildingOverlays(googleMap, getString(R.string.geojson_url));
+        //Providers
+        //TODO Might need to update to more recent version
+        FusedLocationProviderClient fusedLocationProvider = LocationServices.getFusedLocationProviderClient(this);
+        mCameraController = new CameraController(googleMap, mPermissionsGranted, fusedLocationProvider);
+        BuildingInfoWindow mBuildingInfoWindow = new BuildingInfoWindow(getLayoutInflater());
```

Comment:
This change would no longer maker mBuildingInfoWindow a member variable. I think it's safe to remove the "m"

Reply:
fixed